### PR TITLE
[Wallet-111] BTT listing api-docs 

### PR DESCRIPTION
**Pull request by eliu-polo**
* Add new changelog
* Add new currency pairs for BTT
* Fix missing pairs for TRX_ETH and TRX_XRP, and some typos

### CODE REVIEW

File: source/includes/_changelog.md [hunk 1]
```diff
@@ -2,10 +2,13 @@
 
 Recent changes and additions to the Poloniex API.
 
+## 2020-02-06 BTT Listing
+Listing of BitTorrent Token (BTT) and the following market USDT_BTT, TRX_BTT.
```
Comment (jseibel-circle) [inline source/includes/_changelog.md]: `following markets` plural

Comment (eliu-polo) [reply source/includes/_changelog.md]: updated

File: source/includes/_reference.md [hunk 2]
```diff
@@ -381,6 +383,9 @@ Id | Currency Pair
 176 | ETH_REP
 179 | ETH_ZEC
 193 | ETH_ZRX
+267 | TRX_ETH
+268 | TRX_XRP
+271 | TRX_BTT
```
Comment (brbaker) [inline source/includes/_reference.md]: This should be above `TRX_ETH`

Comment (eliu-polo) [reply source/includes/_reference.md]: Will update

**Comment by eliu-polo** [comment]
Will update

**Pull request merged**